### PR TITLE
Change Urban command to get the "best definition"

### PR DESCRIPTION
**Pull request by Sgtpanda**
The current algorithm for the urban command just grabs the first result from the API (which is not always the best), this new algorithm doesn't do anything fancy, it just works out which definition has the highest rating instead.

### CODE REVIEW

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt [hunk 1]
```diff
@@ -44,12 +44,21 @@ class Urban : ICommand {
                     if (defs.length() == 0) {
                         return@thenAccept ctx.send("Couldn't find your search in the dictionary...")
                     }
-                    val def = defs.getJSONObject(0) // maybe random?
-                    if (def.getString("definition").length > 1024) {
-                        val permalink = def.getString("permalink")
+                    var max = defs.getJSONObject(0)
+                    var maxthumbs = max.getInt("thumbs_up")
+                    for (i in 1..(defs.length() - 1)) {
```
Comment (Devoxin) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt]: Any reason why this is `for (i in 1..` and not `for (i in 0..`?

Comment (AlexFlipnote) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt]: Probably used to Lua maybe, where Array starts with 1?

Comment (Sgtpanda) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt]: Unless my understanding is wrong, there's no need to iterate through the zeroth element as the code initially assumes the zeroth element is the maximum, so `i=0` would just compare the zeroth element with itself

Comment (Devoxin) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt]: Ah yeah I see

**Comment by Sgtpanda** [comment]
Any reason this didn't get merged?

**Comment by AlexFlipnote** [comment]
@Devoxin You have reviewed it, was it resolved?

**Comment by Devoxin** [comment]
Been waiting to see if other devs wanna review it

**Pull request merged**